src/mango_cursor_view.erl

```diff
@@ -254,3 +264,10 @@ doc_member(Db, RowProps, Opts) ->
                     Else
             end
     end.
+
+%do this check here as its only in the _all_docs that it returns design docs when we don't want it to
```

Comment:
Not a useful comment. If we needed it somewhere else it'd be somewhere else. Its not so obviously we don't. 🍡